scripts/configure-rdsh.ps1

```diff
@@ -332,40 +333,43 @@ Invoke-Expression "scoop install --global $($ScoopPackages -join ' ')"
 Invoke-Expression "scoop list" -OutVariable ScoopList
 foreach ($Package in $ScoopPackages + @("git", "7zip")) {
     if (-not $ScoopList.Name.Contains($Package)) {
-        Write-Error "ERROR: Failed to install Scoop package: $Package"
+        Write-Verbose "[$(get-date -format o)]: Retrying install of failed Scoop package: $Package"
+        Invoke-RetryCommand -Command "scoop install --global $Package"
```

Comment:
Don't you want to re-check if the install retry succeeded and error if it didn't?

Reply:
The Invoke-RetryCommand will do that automatically. It retries 5 times by default, and if they all fail then it exits with error.